packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2643,39 +2651,59 @@ export class ContainerRuntime
 			}
 			const batchStartCsn = processResult.batchStartCsn;
 			const batch = processResult.messages;
-			const messages: {
+			const processedMessages: {
```

Comment:
Why "processed"? They haven't been processed yet.

...Oh, by the RemoteMessageProcessor?  I'd call it "unpacked" if anything.  Or just messages.